lib/firebase.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:firebase/firebase.dart' as firebase;
+import 'package:dashboard/firebase_constants.dart' as fb_constants;
+import 'package:dashboard/logger.dart';
+
+Logger logger = Logger('firebase.dart');
+
+firebase.Auth get firebaseAuth => firebase.auth();
+List<String> get allowedEmailDomains => fb_constants.allowedEmailDomains;
```

Comment:
I don't think we should be setting these via UI code - these should be set in the firebase database rules